### DISCUSSION TITLE
Is there a way to revert to the previous revision?

### DISCUSSION THREAD

**Discussion by ksudate** [fluxcd/flux2#2799] [Q&A] [answered]
Hi, I have a question regarding rollback.

Is there a way to specify one previous revision when a rollback is needed?
I would expect to be able to instantly Rollback through cli instead of `git revert`.

I would like to know if there are any other common revert procedures.

**Comment by makkes** [comment, marked as answer]
The source (e.g. git or an s3 bucket) acts as the source of truth. That's where you declare the desired state. If you want to go back to a previous revision of the source, you'll have to either `git revert` as you suggested or point the source to e.g. a different branch or revision.
